packages/clerk-js/src/ui/hooks/usePasswordComplexity.ts

```diff
@@ -75,7 +76,13 @@ export const usePasswordComplexity = (config: UsePasswordComplexityConfig, callb
 
   const generateErrorText = useCallback(
     (failedValidations: ComplexityErrorMessages) => {
-      const messageWithPrefix = Object.values(failedValidations).join(', ');
+      let messageWithPrefix: string;
+      if (canUseListFormat(locale)) {
+        const formatter = new Intl.ListFormat(locale, { style: 'long', type: 'conjunction' });
+        messageWithPrefix = formatter.format(Object.values(failedValidations));
+      } else {
+        messageWithPrefix = Object.values(failedValidations).join(', ');
+      }
       return `${t(localizationKeys('unstable__errors.passwordComplexity.sentencePrefix'))} ${messageWithPrefix}`;
```

Comment:
💡  I just noticed this one. Isn't this somewhat inconsistent because it produces the final string out of concatenation between a `localizationKey` and the `failedValidations` values? But I guess this is the only way, due to the dynamic list of failed validations, we cannot pass the values to the localization files, right? 🤔